src/EditorFeatures/Core/EditAndContinue/EditAndContinueLanguageService.cs

```diff
@@ -260,6 +260,7 @@ public async ValueTask EndSessionAsync(CancellationToken cancellationToken)
         UpdateApplyChangesDiagnostics([]);
 
         sourceTextProvider.Deactivate();
+        _debuggingSession?.Dispose();
```

Comment:
this was missed before.

Reply:
EndDebuggingSessionAsync calls already Dispose on the session.

Reply:
updated to just have comment here.